client/my-sites/site-monitoring/components/site-monitoring-line-chart/index.tsx

```diff
@@ -104,42 +130,7 @@ export const SiteMonitoringLineChart = ( {
 						fill: () => '#fff',
 					},
 				},
-				series: [
-					{
-						label: translate( 'Time period' ),
-						value: ( self: uPlot, rawValue: number ) => {
-							// outputs legend content - value available when mouse is hovering the chart
-							if ( ! rawValue ) {
-								return '-';
-							}
-
-							return getPeriodDateFormat(
-								period,
-								new Date( rawValue * 1000 ),
-								getLocaleSlug() || 'en'
-							);
-						},
-					},
-					{
-						fill: solidFill ? fillColor : getGradientFill( fillColor, scaleGradient ),
-						label: translate( 'HTTP requests per sec' ),
-						stroke: '#0675C4',
-						width: 2,
-						paths: ( u, seriesIdx, idx0, idx1 ) => {
-							return spline?.()( u, seriesIdx, idx0, idx1 ) || null;
-						},
-						points: {
-							show: false,
-						},
-						value: ( self: uPlot, rawValue: number ) => {
-							if ( ! rawValue ) {
-								return '-';
-							}
-
-							return numberFormat( rawValue, 0 );
-						},
-					},
-				],
+				series: createSeries( lines ),
```

Comment:
I think we might improve the naming convention when I see this in context. What do you think about passing `series` to `configureSeries`?

```
series: configureSeries( series ),
```

Reply:
that sounds good, I will update, thanks

Reply:
Kat fixed it here https://github.com/Automattic/wp-calypso/pull/80382/commits/e85f5cdabf81e5b79cea5af3bb0fa74fadeef3d9